Fix wordpress documentation regarding mysql non-default port. (#243)
CleverCloud/documentation: content/guides/tutorial-wordpress.md

```

{{< callout type="warning" >}}

> **Make sure you use the port we give you** in `wp-config.php`! Use the `MYSQL_ADDON_PORT` environment variable for that. We do **not** use the default MySQL port.

Reviewer: Added clarity to the sentence in order to define who gives the port, otherwise it can be confusing (could sound like clients need to ask the support for the port, or something like that):

```suggestion
**Make sure you use ** in `wp-config.php` the port the add-on dashboard provides. Use the `MYSQL_ADDON_PORT` environment variable for that. We **don't** use the default MySQL port.
```
